packages/@react-types/actiongroup/src/index.d.ts

```diff
@@ -15,19 +15,32 @@ import {Key} from 'react';
 
 // Not extending CollectionBase to avoid async loading props
 export interface ActionGroupProps<T> extends DOMProps, StyleProps, MultipleSelection {
+  /**
+   * The axis the ActionGroup should align with.
+   * @default 'horizontal'
+   */
   orientation?: Orientation,
   children: ItemElement<T> | ItemElement<T>[] | ItemRenderer<T>,
   items?: Iterable<T>,
   itemKey?: string,
   disabledKeys?: Iterable<Key>,
```

Comment:
Would be nice to have these have a description in the Prop table, but you can have that in a TODO/Jira ticket/followup.